oslo/torch/nn/parallel/data_parallel/zero/fully_sharded_data_parallel.py

```diff
@@ -111,15 +100,14 @@ def __init__(
             hidden_dim=hidden_dim,
             search_range_mb=search_range_mb,
             min_chunk_size_mb=min_chunk_size_mb,
-            strict_ddp_flag=strict_ddp_mode,
         )
         self.heterogeneous_manager = HeterogeneousMemoryManager(
             placement_policy, self.chunk_manager, memstats
         )
         self.force_outputs_fp32 = force_outputs_fp32
         self.param_op_hook = HeterogeneousZeROHook(self.heterogeneous_manager)
```

Comment:
As far as I remember, `HeterogeneousZeROHook` was derived from `DistributedParamOpHook`, but it seems like you removed `param_op_hook.py`. Did it work?

Reply:
Yes, that's correct. I have made a change to directly execute the hooks that were previously driven by the param hook manager. (In the past, this process was triggered by `__torch_function__` of the dist tensor.)

After the modification, it has been confirmed that there are no issues with the functioning.

Reply:
I mean, it doesn't look like the parent class of `HeterogeneousZeROHook` is defined.

Reply:
Thanks, I'll fix it right away. sorry.